### PR TITLE
Update error messaging of nvJPEG

### PR DESCRIPTION
**Pull request by JanuszL**
- add pretty printing for NVJPEG_STATUS_IMPLEMENTATION_NOT_SUPPORTED
  nvJPEG error status added in CUDA 11.6

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:

**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- add pretty printing for NVJPEG_STATUS_IMPLEMENTATION_NOT_SUPPORTED
  nvJPEG error status added in CUDA 11.6

### Affected modules and functionalities:
- error pretty printing in nvjpeg_helper.h 

### Key points relevant for the review:
- NA

### Tests
- [x] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x/] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

File: dali/operators/decoder/nvjpeg/nvjpeg_helper.h [hunk 1]
```diff
@@ -71,6 +71,12 @@ class NvjpegError : public std::runtime_error {
         return "Error during the execution of the device tasks.";
       case NVJPEG_STATUS_IMPLEMENTATION_NOT_SUPPORTED:
         return "Not supported.";
+#if NVJPEG_VER_MAJOR > 11 || \
+    (NVJPEG_VER_MAJOR == 11 && (NVJPEG_VER_MINOR > 6 || \
+                               (NVJPEG_VER_MINOR == 6 && NVJPEG_VER_PATCH >= 0)))
```
Comment (banasraf) [inline dali/operators/decoder/nvjpeg/nvjpeg_helper.h:76]: minor > 6 || (minor == 6 && patch >= 0)
Isn't that just equivalent to:
minor >=6
?

Comment (JanuszL) [reply dali/operators/decoder/nvjpeg/nvjpeg_helper.h]: True. Fixed.

**Review by banasraf — APPROVED** [review]

File: dali/operators/decoder/nvjpeg/nvjpeg_helper.h (hunk 1 above)
Comment (prak-nv) [reply dali/operators/decoder/nvjpeg/nvjpeg_helper.h]: it seems to me it isn't. I read it all nvjpeg of "11.6.x" all minor 11.y.0  ie. releases with patch level 0 after minor 6"

**Review by prak-nv — APPROVED** [review]

File: dali/operators/decoder/nvjpeg/nvjpeg_helper.h (hunk 1 above)
Comment (JanuszL) [reply dali/operators/decoder/nvjpeg/nvjpeg_helper.h]: It was based on the code where the patch version was =! 0, but in this particular case, it is. So Rafal's suggestion should do.

**Pull request merged**